apps/web/src/components/Basenames/RegistrationValueProp/index.tsx

```diff
@@ -0,0 +1,63 @@
+'use client';
+
+import faceScan from './assets/faceScan.svg';
+import currencies from './assets/currencies.svg';
+import sofort from './assets/sofort.svg';
+import globe from './assets/globeWhite.webm';
+import classNames from 'classnames';
+import ImageAdaptive from 'apps/web/src/components/ImageAdaptive';
+import { StaticImageData } from 'next/image';
+
+type ValuePropProps = {
+    title: string;
+    description: string;
+    image: StaticImageData;
+    color: string;
+};
+
+const valueClass = classNames(
+    'relative px-2 py-2',
+  );
+
+function ValueProp({ title, description, image, color }: ValuePropProps) {
+    return (
+        <div className="rounded-xl flex items-center bg-gray-5 py-2 px-2 space-x-2 shadow-md border border-palette-line/20 bg-opacity-80">
+            {/* bg-[#0052FF],blue --- bg-[#22AD73],green --- bg-[#8A55E9], purple */}
+            <div className={`bg-[${color}] rounded-2xl px-6 py-3`}>
+                <ImageAdaptive src={image} alt={title} width={74} height={74} className='flex items-center justify-center' />
+            </div>
+            <div className='px-6'>
+                <h3 className="text-xl mb-1">{title}</h3>
+                <p className="text-gray-600 text-sm">{description}</p>
+            </div>
+        </div>
+      );
+}
+
+export default function RegistrationValueProp() { 
+    return (
+        <section className="flex flex-col items-center max-w-5xl mx-auto pt-24 px-4 md:px-12">
+            <h1 className='text-left z-20 text-[##0A0B0D] text-pretty pl:2 md:text-center text-palette-foreground text-5xl sm:text-6xl'>
```

Comment:
prob want to make sure you've got the text color you want after removing the duplicate #
```suggestion
            <h1 className='text-left z-20 text-[#0A0B0D] text-pretty pl:2 md:text-center text-palette-foreground text-5xl sm:text-6xl'>
```